test/CQRS/LeanCode.CQRS.MassTransitRelay.Tests/Middleware/EventsPublisherMiddlewareWithCustomConfigurationTests.cs

```diff
@@ -0,0 +1,196 @@
+#nullable enable
+using System.Net.Http.Headers;
+using System.Security.Claims;
+using FluentAssertions;
+using LeanCode.Contracts;
+using LeanCode.CQRS.Execution;
+using LeanCode.CQRS.MassTransitRelay.Middleware;
+using LeanCode.DomainModels.Model;
+using LeanCode.IntegrationTestHelpers;
+using MassTransit;
+using MassTransit.Testing;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Xunit;
+
+namespace LeanCode.CQRS.MassTransitRelay.Tests.Middleware;
+
+public sealed class EventsPublisherMiddlewareWithCustomConfigurationTests : IDisposable, IAsyncLifetime
```

Comment:
This is exactly the same test suite as `EventsPublisherMiddlewareTests`. Can we not duplicate the code?